micro-rdk/src/common/credentials_storage.rs

```diff
@@ -70,6 +58,21 @@ impl From<RobotCredentials> for CloudConfig {
     }
 }
 
+#[derive(Clone, Debug, Default)]
```

Comment:
Was there a particular reason to move this?

Reply:
to group `WifiCredentials` and `WifiCredentialsStorage` closer together. though looking at everything it might be better to move all wifi credentials stuff to the beneath `RobotConfigurationStorage`. can do it before merging.